app/routes.js

```diff
@@ -8,139 +8,108 @@ import { errorHandler } from './pages/error/errorHandler';
 
 const router = express.Router();
 
+const withCatch = controller => async (req, res, next) => {
```

Comment:
so this withCatch wraps around the innards of the routes. Is that right? I just have a slight issue with it being called `controller`. Since we have something else that we call the controller.

Reply:
Yes - what do you suggest? I couldn't think of anything at the time

Reply:
route?

Reply:
lol lovely 👍

Reply:
😎